modules/shutter_page/templates/custom_endpoint.json

```diff
@@ -0,0 +1,99 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "profile_name": {
+            "defaultValue": "hmcts-shutter-page-nonprod",
+            "type": "String"
+        },
+        "endpoint_name": {
+            "type": "String"
+        },
+        "originHostHeader": {
+            "type": "String"
+        },
+        "originPath": {
+            "type": "String"
+        },
+        "originStorageBlob": {
+            "type": "String"
+        }
+    },
+    "variables": {
+        "endpoint": "[concat(parameters('profile_name'), '/', parameters('endpoint_name'))]",
+        "storageBlob": "[replace(parameters('originStorageBlob'),'.', '-')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Cdn/profiles/endpoints",
+            "apiVersion": "2019-04-15",
+            "name": "[variables('endpoint')]",
+            "location": "WestUs",
```

Comment:
same q here, does it only work in west us?

Reply:
answered above